src/components/my-reading/reviewing-article-section.js

```diff
@@ -103,8 +102,8 @@ const ReviewingArticleSection = () => {
       } else {
         setShowingReviewingArticle(formattedReviews)
       }
-      setIsLoading(false)
     })
+    setIsLoading(false)
```

Comment:
Question: do we need to wait for fetching data done (finally) before setting `isLoading` false ?

Reply:
fixed